charts/karpenter/templates/webhook/webhooks.yaml

```diff
@@ -50,3 +50,19 @@ webhooks:
     - CREATE
     - UPDATE
     - DELETE
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: validation.webhook.config.karpenter.sh
```

Comment:
Maybe this is the magic of knative, but why do we now need to validate configmaps? Is it possible for any user to change the configmap that wasn't already done before, or did we just not have this as needed?

Reply:
We're validating the logging configuration.

Reply:
After reading it more I understand that now. Thanks.